discover_role_repos.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+for defaults_file in roles/*/defaults/main.yml ; do
+  role="$(echo "${defaults_file}" | cut -f2 -d'/')"
+  yq eval "[{\"repo\": .${role}_repo, \"role\": \"${role}\"}]" "${defaults_file}"
+done | yq -o json -I=0
```

Comment:
Shouldn't this be `vars/main.yml` now that the repo var lives there?

Reply:
Yes, forgot to update that.